stdlib/public/core/StringCreate.swift

```diff
@@ -312,9 +312,19 @@ extension String {
   @inline(never) // slow-path
   internal static func _copying(_ str: Substring) -> String {
     if _fastPath(str._wholeGuts.isFastUTF8) {
-      return unsafe str._wholeGuts.withFastUTF8(range: str._offsetRange) {
+      var new = unsafe str._wholeGuts.withFastUTF8(range: str._offsetRange) {
```

Comment:
In non-watchOS or 64-bit environments, will the compiler not complain about this `var` being unmutated?

Reply:
I didn't notice it complaining when building on macOS.